app/shared/resource-list/ResourceListItem.js

```diff
@@ -26,7 +28,20 @@ class ResourceListItem extends Component {
             height={420}
             image={getMainImage(resource.images)}
             width={700}
-          />
+          >
+            <Label
+              className="app-ResourceListItem__peopleCapacity"
+              shape="rounded"
```

Comment:
- [ ] Here shape "circle" could be used so there would be less need for style customizations.